cost-optimization/gke-keda/cloud-pubsub/cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file and other cloudbuild.yaml files are used to ensure that
+# our public Docker images such as us-docker.pkg.dev/google-samples/containers/gke/pubsub-sample:v1
+# are rebuilt and updated upon changes to the repository.
+
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args:
+    - 'build'
+    - '-t'
+    - 'gcr.io/google-samples/keda-pubsub-sample:v1'
+    - '-t'
+    - 'gcr.io/google-samples/keda-pubsub-sample:sample-public-image-v1-${SHORT_SHA}'
+    - '-t'
+    - 'us-docker.pkg.dev/google-samples/containers/gke/keda-pubsub-sample:v1'
+    - '-t'
+    - 'us-docker.pkg.dev/google-samples/containers/gke/keda-pubsub-sample:sample-public-image-v1-${SHORT_SHA}'
+    - '.'
```

Comment:
GCR is deprecated. Since this is a new sample and we can "start fresh", why not simply tag with an AR path (`us-docker.pkg.dev/` and not tag or use `gcr.io` at all? (In other words, remove the top two image tags)

Reply:
Done. (My sample relies on AR only.)